keyboards/kalkulators/info.json

```diff
@@ -0,0 +1,50 @@
+{
+    "manufacturer": "georgsnarbuts",
+    "keyboard_name": "kalkulators",
+    "maintainer": "georgsnarbuts",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["A6", "A5", "A4", "A3"],
+        "rows": ["A7", "B0", "A2", "A1", "A0"]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x4B4C",
+        "vid": "0x474E",
+        "no_startup_check": true
+    },
+    "layouts": {
+        "LAYOUT_calc": {
```

Comment:
```suggestion
        "LAYOUT": {
```

Reply:
`LAYOUT_numpad_5x4` would be more appropriate